src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Utilities/TypeStyle/CSharpTypeStyleHelper.cs

```diff
@@ -108,8 +108,7 @@ SyntaxKind.ForStatement or
                 SyntaxKind.UsingStatement);
 
             return isSupportedParentKind &&
-                variableDeclaration.Variables.Count == 1 &&
-                variableDeclaration.Variables.Single().Initializer.IsKind(SyntaxKind.EqualsValueClause);
+                variableDeclaration.Variables is [{ Initializer: not null }];
```

Comment:
Missing the kind check? Is that the only possible kind?

Reply:
Yeah, this is a redundant kind check.  Initializer can only be an EqualsValueClause, so this is really just a not-null check.